cloud/aws/templates/aws_oidc/vpc.tf

```diff
@@ -2,10 +2,26 @@
 
 data "aws_availability_zones" "available" {}
 
+locals {
+  enable_managed_vpc = alltrue([
```

Comment:
**nitpick:** Could consider adding a comment that all fields of the external_vpc variable need to be set for it to be used.